data/Mohiit70.json

```diff
@@ -11,17 +11,17 @@
     {
       "name": "Follow me on LinkedIn",
       "url": "https://www.linkedin.com/in/mohit-bisht70/",
-      "icon": "FaLinkedIn"
+      "icon": "FaLinkedin"
     },
     {
       "name": "Follow me on GitHub",
       "url": "https://github.com/Mohiit70",
-      "icon": "FaGitHub"
+      "icon": "FaGithub"
     },
     {
       "name": "Follow me on hashnode",
       "url": "https://mohitbisht70.hashnode.dev/",
-      "icon": "Fahashnode"
+      "icon": "FaHashnode"
```

Comment:
Hi @Mohiit70, thank you for contributing  :tada: 
Please replace this with  **' SiHashnode '** 
You can checkout [ICONS](https://linkfree.eddiehub.io/icons) to search icons.  :+1: